is_iot_sink/irrigation/scheduled/scheduled_irrigation.py

```diff
@@ -0,0 +1,57 @@
+import datetime
+
+from is_iot_sink.allowed_collectors import AllowedCollectors
+from is_iot_sink.irrigation.irrigation_mode import *
+from is_iot_sink.irrigation.automated.flc import flc
+from is_iot_sink.irrigation.valves.valves_manager import ValveManager
+from is_iot_sink.mongodb.mongodb_client import MongoClient
+from is_iot_sink.settings import Settings
+from is_iot_sink.logger import LOG
+import time
+import threading
+
+
+class ScheduledIrrigation:
+    def __init__(self, valve_manager: ValveManager, mongo_client: MongoClient):
+        super().__init__()
+        self.__valve_manager = valve_manager
+        self.__mongo_client = mongo_client
+        self.mode = IrrigationMode.SCHEDULED
+        self.running = False
```

Comment:
why are these 2 not marked as private as well, as they don't seem to be modified or used ourside the class

Reply:
Good point, missed them. 

The irrigation mode should have a getter. We want from the Sink application to check the actual irrgation mode.
https://github.com/luciiii6/is-iot-sink/pull/3/files#diff-316c0baad536662e1332877cec5b8aff4cc321f81f82a67ae6019e88d5c28872R44